<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>601</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>7</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_4">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>24</number>
        </property>
        <property name="rightMargin">
         <number>1</number>
        </property>
        <property name="bottomMargin">
         <number>11</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Camera Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cmbCamType">
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>21</width>
            <height>22</height>
           </size>
          </property>
          <item>
           <property name="text">
            <string>Vision Sensor</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Vision System</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Ip Address</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtIpAdr">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Port Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spPortT">
          <property name="value">
           <number>23</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>User</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtUser">
          <property name="text">
           <string>admin</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Password</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtPsswrd">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbConnect">
          <property name="text">
           <string>Connect To Camera</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QFormLayout" name="lyCmdPr">
            <property name="verticalSpacing">
             <number>63</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>Send</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>command status</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>message</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Console View</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="text">
             <string>Export to...</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="treeWidget">
          <column>
           <property name="text">
            <string>Commands</string>
           </property>
          </column>
          <item>
           <property name="text">
            <string>Basic</string>
           </property>
           <item>
            <property name="text">
             <string>Execution &amp; Online</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Settings &amp; Cell Value </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Image </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>File and Jobs</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Extended</string>
           </property>
           <item>
            <property name="text">
             <string>Put</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Get</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>EV</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string>Special Procedures</string>
           </property>
           <item>
            <property name="text">
             <string>Fix Camera Name</string>
            </property>
           </item>
          </item>
          <item>
           <property name="text">
            <string/>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
